whitepaper/chapters/blockchain_sync_flowchart.tex

```diff
@@ -1,3 +1,4 @@
+\index{synchronization}
 \index{block!synchronization}
```

Comment:
is there reason for double index ?

Reply:
yes, to have it both under `block |- synchronization` and separately as `synchronization`